app/models/ext_management_system.rb

```diff
@@ -634,7 +634,7 @@ def build_endpoint_by_role(options)
     endpoint = endpoints.detect { |e| e.role == options[:role].to_s }
     # update or create
     if endpoint
-      endpoint.update(options)
+      endpoint.assign_attributes(options)
```

Comment:
Please remove the comment here or in a follow up PR.

https://twitter.com/misterjack/status/603453117179555840
https://twitter.com/nzkoz/status/538892801941848064